base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -397,6 +404,7 @@ impl ConsensusConstants {
             faucet_value: 0.into(),
             transaction_weight: TransactionWeight::latest(),
             max_script_byte_size: 2048,
+            max_encrypted_data_byte_size: 32, // U256 size
```

Comment:
This should be the current `EncryptedData::data: [u8; SIZE_TOTAL]` plus 256 bytes,